get/releases_test.go

```diff
@@ -0,0 +1,248 @@
+package get
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"testing"
+	"time"
+
+	runtimev1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	apps "github.com/ninech/apis/apps/v1alpha1"
+	meta "github.com/ninech/apis/meta/v1alpha1"
+	"github.com/ninech/nctl/api"
+	"github.com/ninech/nctl/api/util"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var (
+	// TODO: would be nice to get this from "github.com/ninech/apis/apps/v1alpha1"
+	AppMicro     apps.ApplicationSize = "micro"
+	AppMini      apps.ApplicationSize = "mini"
+	AppStandard1 apps.ApplicationSize = "standard-1"
+
+	StatusSuperseded apps.ReleaseProcessStatus = "superseded"
+	StatusAvailable  apps.ReleaseProcessStatus = "available"
+
+	defaultConfig = apps.Config{
+		Size:     &AppMicro,
+		Replicas: pointer.Int32(1),
+		Port:     pointer.Int32(8080),
+	}
+
+	defaultCreationTime = metav1.NewTime(mustParseTime(time.RFC3339, "2023-03-13T14:00:00Z"))
+)
+
+func mustParseTime(format string, value string) time.Time {
+	t, _ := time.Parse(format, value)
+	return t
```

Comment:
Shouldn't `must*` functions usually panic when running into an error? This is more of a `maybeParseTime` 😄